lib/Target/CBackend/CBackend.cpp

```diff
@@ -2547,7 +2547,8 @@ void CWriter::generateHeader(Module &M) {
     Out << "static __forceinline ";
     printTypeName(Out, RTy, isSigned);
     const auto Pred = (*it).first;
-    if (CmpInst::isFPPredicate((*it).first)) {
+    if (CmpInst::isFPPredicate(Pred)) {
+      headerUseFCmpOp(Pred);
       FCmpOps.insert(Pred);
```

Comment:
```suggestion
```

Reply:
Can you elaborate on why row 2552 is not needed in this case?

Reply:
It's what `headerUseFCmpOp` does

Reply:
Ah, yes, I realized that after I wrote above... :)